src/pip/_internal/cli/req_command.py

```diff
@@ -224,7 +224,7 @@ def warn_if_run_as_root() -> None:
         "Running pip as the 'root' user can result in broken permissions and "
         "conflicting behaviour with the system package manager. "
         "It is recommended to use a virtual environment instead: "
-        "https://pip.pypa.io/warnings/venv"
+        "https://pip.pypa.io/en/stable/cli/pip_install/#installing-packages-as-root"
```

Comment:
Don't change this -- the URL being used is a redirect, that can be updated manually by a maintainer.